generator/internal/dart/dart.go

```diff
@@ -139,3 +142,19 @@ func shouldGenerateMethod(m *api.Method) bool {
 	// TODO(#499) Switch to explicitly excluding such functions.
 	return !m.ClientSideStreaming && !m.ServerSideStreaming && m.PathInfo != nil && len(m.PathInfo.PathTemplate) != 0
 }
+
+func formatDirectory(dir string) error {
+	return runExternalCommand("dart", "format", dir)
```

Comment:
Consider something like this:

https://github.com/googleapis/google-cloud-rust/blob/6c64aa5bf5ce331c0e47ec2d488008ed93e7ca5e/generator/internal/sidekick/rust_generate.go#L58-L60

From time to time a random developer in the SDK team may need to run these tools and they would appreciate a link on how to install the toolchain.

Reply:
will do